.circleci/config.yml

```diff
@@ -328,7 +328,7 @@ jobs:
           at: /tmp/generated-sql
       - run:
           name: Move generated-sql into place
-          command:
+          command: |
```

Comment:
This was being interpreted as a single `rm -rf` invocation removing such illustrious directories as `cp` and `-r`.